tests/trainers/tensorflow_examples.py

```diff
@@ -0,0 +1,72 @@
+"""Basic tests for the Python base container images,based on tensorflow library.
+Find a description in the README file
+"""
+
+import tensorflow as tf
+
+
+def tensorflow_example_1():
+    # Train a machine learning model using a prebuilt dataset with the Keras API.
+    #
+    # Original tutorial from the TensorFlow 2 quickstart for beginners link:
+    # https://github.com/tensorflow/docs/blob/master/site/en/tutorials/quickstart/beginner.ipynb
+    #
+    # @title Licensed under the Apache License, Version 2.0 (the "License");
+    # you may not use this file except in compliance with the License.
+    # https://www.apache.org/licenses/LICENSE-2.0
+    #
+
+    print("TensorFlow version:", tf.__version__)
+
+    # Load a dataset
+    mnist = tf.keras.datasets.mnist
+
+    (x_train, y_train), (x_test, y_test) = mnist.load_data()
+    x_train, x_test = x_train / 255.0, x_test / 255.0
+
+    # Build a machine learning model
+    model = tf.keras.models.Sequential(
+        [
+            tf.keras.layers.Flatten(input_shape=(28, 28)),
+            tf.keras.layers.Dense(128, activation="relu"),
+            tf.keras.layers.Dropout(0.2),
+            tf.keras.layers.Dense(10),
+        ]
+    )
+
+    #  the model returns a vector of logits or log-odds scores, one for each class
+    predictions = model(x_train[:1]).numpy()
+
+    # converts these logits to probabilities for each class:
+    tf.nn.softmax(predictions).numpy()
+
+    # Define a loss function for training
+    loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)
+
+    loss_fn(y_train[:1], predictions).numpy()
```

Comment:
Does this line actually do anything? At least the `numpy()` part is definitely superfluous.

Reply:
This line shows the initial value of the loss that is > 2.0, meaning almost random evaluation, and after the optimization it will be closer to zero (<0.1); numpy() method gets the pure value (array or scalar) from the Tensor object.
I suggest we could collect this initial value to validate that here it was not optimized yet, against the final check.

Reply:
Code updated, see line 70